test-js/unit/prompting/PromptPanelSpec.js

```diff
@@ -546,14 +546,20 @@ define([ 'dojo/number', 'dojo/i18n', 'common-ui/prompting/PromptPanel',
             }
           });
           comp.type = "ScrollingPromptPanelLayoutComponent";
-          spyOn(window, "$");
+
+          // TODO: Find alternative for:
+          //spyOn(window, "$");
```

Comment:
So, for now, I'd accept this as is (commented with TODO), and later you guys can replace it with some alternative to testing what this is testing. What do you think?